bunq/sdk/exception.py

```diff
@@ -1,44 +1,22 @@
-class ApiException(Exception):
-    """
-    :type _response_code: int
-    """
-
-    # Constants for formatting messages
-    _FORMAT_RESPONSE_CODE_LINE = 'HTTP Response Code: {}'
-    _GLUE_ERROR_MESSAGES = '\n'
-
-    def __init__(self, response_code, messages):
+class BunqError(Exception):
```

Comment:
Let's call the base class `ApiException` and then extend from it. For the case of an unknown exception, we can throw the `ApiException` itself or make something like `UnknownApiErrorException`!